src/integTest/groovy/com/bmuschko/gradle/docker/ToolingApiIntegrationTest.groovy

```diff
@@ -60,4 +60,8 @@ repositories {
     protected String createUniqueImageId() {
         "gradle/${UUID.randomUUID().toString().replaceAll('-', '')}"
     }
+
+    protected String createUniqueContainerName() {
```

Comment:
Sure, I will refactor that. Didn't want to interfere too much with the other test code.